sanctuarycomputer/contentful-fragment
# Bugfix - add/remove items from muuri grid

Add callbacks so that muuri grid will update when entries are added/removed. 
**This fixes...**
 an issue where adding a new entry would cause it to be positioned behind the existing grid, rather than becoming part of the existing grid.

**But, not yet perfect...** 
Need to force grid to recalculate the height of all items + re-render when editingUUID changes.
Something like 
```
updateGrid: function() {
  get(this, 'grid').refreshItems();
}.observes('editingUUID')
```
But ["refreshItems"](https://github.com/haltu/muuri#gridrefreshitems-items-) is not behaving as expected so this issue is still outstanding.

To Test: 
set `const DEV = true;` in application.js & navigate to localhost:4200. Confirm that functionality is improved.
